modules/text/src/precomp.hpp

```diff
@@ -48,6 +48,9 @@
 #include "text_config.hpp"
 
 #ifdef HAVE_TESSERACT
+#if !defined(USE_STD_NAMESPACE)
+#define USE_STD_NAMESPACE
```

Comment:
@LaurentBerger, using that macro is the new correct way to solve the problem with `string`. I'm afraid that Google's coding conventions are something which we cannot change. The usage of `std::` is inconsistent, yes.

In addition to the macro, I'd also add a comment which explains why the macro is needed. Maybe some day it won't be needed any more.